src/test/java/org/wildfly/security/ldap/X509EvidenceVerificationSuiteChild.java

```diff
@@ -44,6 +49,19 @@
  */
 public class X509EvidenceVerificationSuiteChild {
 
+    private static final String LDAP_DIRECTORY_LOCATION = "./target/test-classes/ldap";
+    private static final String LDIF_LOCATION = "/elytron-x509-verification.ldif";
+
+    @AfterClass
```

Comment:
I think this clean up should be done after each test since both can write to the LDIF file.

Reply:
Fixed